openstef/data_classes/prediction_job.py

```diff
@@ -81,6 +82,10 @@ class PredictionJobDataClass(BaseModel):
     """Only required for create_wind_forecast task"""
     hub_height: Optional[float]
     """Only required for create_wind_forecast task"""
+    pipelines_to_run: PipelineType = PipelineType.BOTH
```

Comment:
What do you think of implementing this as a list of PipelineTypes? This way we do not have to define a separate enum catergory for each pipeline combination we want to run.

Reply:
Yes, It's probably better with a list. I follow your suggestion.